docker/Dockerfile

```diff
@@ -48,8 +48,8 @@ RUN git clone https://github.com/cocodataset/cocoapi.git \
  && python setup.py build_ext install
 
 # install PyTorch Detection
-RUN git clone https://github.com/bendidi/maskrcnn-benchmark.git \
+RUN git clone https://github.com/facebookresearch/maskrcnn-benchmark.git \
  && cd maskrcnn-benchmark \
- && FORCE_CUDA=1 python setup.py build develop
```

Comment:
please do it like this:

```
# install PyTorch Detection
ARG FORCE_CUDA="1"
ENV FORCE_CUDA=${FORCE_CUDA}
RUN git clone https://github.com/facebookresearch/maskrcnn-benchmark.git \
 && cd maskrcnn-benchmark \
 && python setup.py build develop
```

build args are limited to the scope of the build and are not environment variables, they are intended to change the behavior of the build.